### ISSUE TITLE
Could not find module `ember-component`

### ISSUE THREAD

**Issue by devtoro** [zestia/ember-select-box#2]
Hello everyone!

I am working on an ember-cli application ( ember-cli version: 0.2.7 ). When I try to use the addon according to your examples, I get the following error:

_Could not find module `ember-component` imported from `ember-cli-select-box/components/select-box/native`_

I just installed the component and used it as in the example

```
{{#select-box/native as |sb|}}
  {{sb.option value=1 label='One'}}
  {{sb.option value=2 label='Two'}}
  {{sb.option value=3 label='Three'}}
{{/select-box/native}}
```

with no additions. I Tried to "fix" this by importing "Ember" in the component end extend Ember.Component ( which worked ), but then the next "import" failed with the same kind of error.

Any ideas/suggestions? 

Thank you for your time :)

**Comment by amk221** [comment]
Hi there, thanks for the feedback
Are you >= Ember 2.3.0

**Comment by devtoro** [comment]
No, actually I just checked the one issue you had ( status - closed ) and I saw the minimal Ember version needed. Please add that in the readme :)

**Comment by amk221** [comment]
It is on a badge on the readme, although I just corrected it from '>2.3.0` to '>=2.3.0'
Thanks!

**Comment by devtoro** [comment]
Indeed, my bad :)
